go.mod

```diff
@@ -17,8 +17,8 @@ require (
 	k8s.io/apimachinery v0.19.1
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/code-generator v0.19.1
-	kubevirt.io/client-go v0.29.0
-	kubevirt.io/kubevirt v0.29.0
+	kubevirt.io/client-go v0.32.0
+	kubevirt.io/kubevirt v0.32.0
```

Comment:
need to also update kubevirt in [cluster-up script](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/7600ea1e8a945a5665071bdc7e6d8619adb65195/cluster/up.sh#L28)